VideoWeb/VideoWeb/ClientApp/src/app/services/hearing-venue-flags.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
```

Comment:
If committing the change below:
```suggestion
import { ReplaySubject } from 'rxjs';
```